src/Servers/IIS/IIS/src/WebHostBuilderIISExtensions.cs

```diff
@@ -30,7 +30,8 @@ public static IWebHostBuilder UseIIS(this IWebHostBuilder hostBuilder)
             // Check if in process
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && NativeMethods.IsAspNetCoreModuleLoaded())
             {
-                hostBuilder.CaptureStartupErrors(true);
+                // Startup Errors are now reported by ANCM directly.
+                hostBuilder.CaptureStartupErrors(false);
```

Comment:
Should we just *not* call it? If the user specifies `CaptureStartupErrors(true)` on the host builder **before** `UseIIS`, this will stomp that out right? Maybe that's fine given that `CreateDefaultBuilder` sets things up right.

Reply:
Yeah, better to remove the call.

Reply:
CreateDefaultBuilder doesn't set `CaptureStartupErrors`, so I also agree.